src/app/tests/suites/certification/Test_TC_ACT_3_1_Simulated.yaml

```diff
@@ -27,9 +27,21 @@ config:
     endpoint: 0
 
 tests:
+    #- label: "Wait for the device to be commissioned"
+    #  cluster: "DelayCommands"
+    #  command: "WaitForCommissioning"
     - label: "Wait for the device to be commissioned"
-      cluster: "DelayCommands"
-      command: "WaitForCommissioning"
+      PICS: PICS_SKIP_SAMPLE_APP
```

Comment:
@manjunath-grl need to remove the entire first step including the label and user prompt

Reply:
Removed the prompt step.